source/technical/reservations.rst

```diff
@@ -96,15 +96,17 @@ up the window displayed below:
 	
 	   The Create Lease dialog Host reservation tab
 
-#. Choose the minimum and maximum number of hosts.
+    a. Check "Reserve Hosts".
 
-#. Choose a node type in the drop down menu below the *node_type* and *=* drop down lists.
+    b. Choose the minimum and maximum number of hosts.
 
-   .. note::
+    c. Choose a node type in the drop down menu below the *node_type* and *=* drop down lists.
+
+       .. note::
 
-      You may only request one type of node in each individual lease. If you
-      wish to request multiple node types, you must create separate Leases for
-      each node type.
+          You may only request one type of node in each individual lease. If you
```

Comment:
It'd be interesting if we eventually supported this; I think the new UI is more scalable and could.